features/admin_associates_customer_with_auction.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Admin associates customer with auction
+  As an administrator
+  I want to be able to associate auctions with customers
+  So that users and admins can see who is creating certain projects
+
+  Scenario: Associating an auction with a customer
```

Comment:
was thinking more like a feature spec file that is related to more generally "creating auctions" or "updating auctions" -- your call!